### PR TITLE
Integration1 t6 f2 15

### PR DESCRIPTION
**Pull request by nala-he**
I made the following revisions based on the feedback in the closed pull request#3: 1. changed open hours to hours, followed yelp business open hours format, 
2. deleted "closed on" in business info section, 
3. changed the Route path to the business-home component into "/restaurant/:rid" in tuiter/index.js.

### CODE REVIEW

File: src/components/restaurant/business-home.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                     className="wd-border-radius wd-banner-width mt-3" height={200}/>
+                <img className="wd-portrait position-absolute start-0 ms-5"
+                     src={`/images/${restaurant.profilePicture}`}/>
+            {/*    <Link to="../edit-restaurant"*/}
+            {/*          className="btn btn-white border rounded-pill fw-bolder position-relative*/}
+            {/*float-end me-3 mt-2">*/}
+            {/*        Edit</Link>*/}
+            </div>
+            <div className="m-3 position-relative">
+                <span className="h5 fw-bolder">{restaurant.name} {restaurant.handler}
```
Comment (MaceeQi) [inline src/components/restaurant/business-home.js]: "handler" --> "handle"

File: src/components/restaurant/business-home.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import FeaturedItems from "./featured-items";
+
+const BusinessHome = (
+{
+    restaurant = {
+        _id: "123",
+        name: "Red Lobster",
+        bannerPicture: "redlobster-banner.jpeg",
+        profilePicture: "redlobster-avatar.png",
+        handler: "@redlobster",
```
Comment (MaceeQi) [inline src/components/restaurant/business-home.js]: I think we should change "handler" to "handle" - small detail for now but just for consistency when we create the database so we don't have to come back to refactor any of the property names

**Review by MaceeQi — COMMENTED** [review]
Just a small suggestion to change the property "handler" to "handle". Otherwise, everything looks good :)

File: src/components/restaurant/business-home.js (hunk 2 above)
Comment (nala-he) [reply src/components/restaurant/business-home.js]: Got it! Solved.

File: src/components/restaurant/business-home.js (hunk 1 above)
Comment (nala-he) [reply src/components/restaurant/business-home.js]: Thank you for pointing it out! Revised.

**Review by t1yena — APPROVED** [review]

**Comment by t1yena** [comment]
everything looks good!

**Pull request merged**